delta/ml/ml_config.py

```diff
@@ -272,6 +272,28 @@ def dir(self) -> str:
             tbd = os.path.join(appdirs.AppDirs('delta', 'nasa').user_data_dir, 'tensorboard')
         return tbd
 
+class ClassifyConfig(config.DeltaConfigComponent):
+    """
+    Configure classification options.
+    """
+    def __init__(self):
+        super().__init__()
+        self.register_field('metrics', list, 'metrics', None, 'List of metrics to apply.')
```

Comment:
Document in config/README.md

Reply:
These are not implemented yet, probably best to save them and results_file for another PR.

Reply:
Sounds good!